client/components/TestRun/index.jsx

```diff
@@ -33,28 +33,30 @@ import './TestRun.css';
 const createGitHubIssueWithTitleAndBody = ({
     test,
     testPlanReport,
-    conflictsFormatted,
-    isReportViewer = false
+    conflictsFormatted
 }) => {
-    const issueRaiser = isReportViewer ? 'Report Viewer' : 'Tester';
-    const title = `${issueRaiser}'s Issue Report for "${test.title}"`;
-
-    const { testPlanTarget } = testPlanReport;
+    const { testPlanVersion, testPlanTarget } = testPlanReport;
     const { at, browser } = testPlanTarget;
+
+    const title =
+        `Feedback: "${test.title}" (${testPlanVersion.title}, ` +
+        `Test ${test.rowNumber})`;
+
     const shortenedUrl = test.renderedUrl.match(/[^/]+$/)[0];
 
     let body =
-        `### Test File at Exact Commit\n\n` +
-        `[${shortenedUrl}](https://aria-at.w3.org${test.renderedUrl})\n\n` +
-        `### AT\n\n` +
-        `${at.name} (version ${testPlanTarget.atVersion})\n\n` +
-        `### Browser\n\n` +
-        `${browser.name} (version ${testPlanTarget.browserVersion})\n\n` +
-        `### Description\n\n` +
-        `_type your description here_`;
+        `#### Description of Behavior\n\n` +
+        `<!-- write your description here -->\n\n` +
+        `#### Test Setup\n\n` +
+        `- Test File at Exact Commit: ` +
+        `[${shortenedUrl}](https://aria-at.w3.org${test.renderedUrl})\n` +
+        `- AT: ` +
+        `${at.name} (version ${testPlanTarget.atVersion})\n` +
+        `- Browser: ` +
+        `${browser.name} (version ${testPlanTarget.browserVersion})\n`;
 
     if (conflictsFormatted) {
```

Comment:
It appears that once a test exists in the suite with conflicts, it is also included in the Issue's output regardless of whether or not the current test which an issue is being raised for has any conflicts.

Reply:
Based on the 11/18 CG meeting, it seems that this is already known based on https://github.com/w3c/aria-at-app/issues/350. Will change my review to approval as that issue isn't related to this PR.

Reply:
Thanks for mentioning that! I'll definitely be fixing this in the conflicts improvement work.